app/controllers/survey/2022.js

```diff
@@ -0,0 +1,662 @@
+import Controller from '@ember/controller';
+import {
+  // AreaSplineChart,
+  // HorizontalBarChart,
+  PieChart,
+  VerticalBarChart,
+} from 'ember-website/utils/highcharts';
+
+// TODO: 2022 Color Palette
+// eslint-disable-next-line no-unused-vars
+const darkGrayColor = '#4b4b4b';
+// Ember Branding Color Palette
+const emberColors = {
+  ember: '#E04E39',
+  burntEmber: '#9B2918',
+  blue: '#1E719B',
+  lightBlue: '#74B0CE',
+  yellow: '#FBC840',
+  black: '#212121',
+  gray: '#817F7F',
+  faintGray: '#EFEBEA',
+  linen: '#FDF7F6',
+  white: '#FDFDFD',
+};
+
+// const orangePalette = {
+//   orange: '#e04e39',
+//   orangeDarker: '#c7341f',
+//   orangeDarkest: '#9b2918',
+//   orangeLighter: '#e77565',
+// };
+
+// Default Color Assignments for Yearly Graph Data Series
+// var color2018 = emberColors.gray;
+// var color2019 = emberColors.blue;
+// var color2020 = orangePalette.orangeDarkest;
+var color2022 = emberColors.burntEmber;
+// eslint-disable-next-line no-unused-vars
+// var colorFutureYear = emberColors.yellow;
+
+const stackColors = {
+  htmlDark: '#E45126',
+  htmlLight: '#F2682A',
+  cssDark: '#0D72B8',
+  cssLight: '#32AADE',
+  jsDark: '#E4A228',
+  jsLight: '#F1BF28',
+  ariaDark: '#1A5E9A',
+};
+
+// Begin Chart Definitions
+
+const S01_Q01_SS__TIME_USE = new VerticalBarChart({
+  chart: {
+    title: `How long have you been using Ember?`,
+    categories: [
+      `I haven't used Ember yet`,
+      `Less than 1 year`,
+      `Between 1 - 3 years`,
+      `Between 4 - 6 years`,
+      `Between 7 - 9 years`,
+      `Greater than 10 years`,
+    ],
+  },
+  rawData: [
+    {
+      color: color2022,
+      label: '2022',
+      values: [0.68, 8.16, 17.52, 34.86, 27.04, 10.37],
+    },
+  ],
+});
+
+const S01_Q02_MS__OTHER_FRAMEWORKS = new VerticalBarChart({
+  chart: {
+    title: `In addition to Ember, with which of these frameworks do you have some experience?`,
+    categories: [`Angular`, `Astro`, `Next`, `Nuxt`, `SvelteKit`],
+  },
+  rawData: [
+    {
+      color: color2022,
+      label: '2022',
+      values: [37.28, 1.77, 22.79, 7.77, 7.07],
+    },
+  ],
+});
+const S01_Q03_MS__OTHER_UI_LIBS = new VerticalBarChart({
+  chart: {
+    title: `Glimmer is Ember's UI Library. What other libraries have Ember users tried?`,
+    categories: [`Preact`, `React`, `Solid`, `Svelte`, `Vue`],
+  },
+  rawData: [
+    {
+      color: color2022,
+      // TODO label should be optional since we don't always need both label and title
+      label: '',
```

Comment:
```suggestion
      label: '2022',
```

In this pull request, let's provide the value `'2022'`. Otherwise, the chart will look funny:

<img width="1060" alt="We see a red circle without any label" src="https://user-images.githubusercontent.com/16869656/164542832-e963f825-96cf-417c-b054-9c95908cd1c4.png">

In a separate issue and pull request, we can hide the label when `rawData` contains 1 array element.

Reply:
@josephdsumner @MelSumner

I think Highcharts is deciding whether to show the legends for a particular type of chart (i.e. we don't have control over it at the moment). The [utility for the vertical bar chart](https://github.com/ember-learn/ember-website/blob/main/app/utils/highcharts/vertical-bar-chart.js) doesn't show a related key in the object.

I'd not make the feature request a blocker for this pull request. Let's open an issue and make a separate pull request so that the person working on it can also add unit tests?

Reply:
In #935, I added the feature to conditionally hid the chart legend. (Later today, I'll describe the pull request in detail, then mark the pull request as ready for review.)